tests/DFE.FindInformationAcademiesTrusts.CypressTests/cypress/e2e/regression/trusts/financial-docs.cy.ts

```diff
@@ -0,0 +1,177 @@
+            });
+
+            it("Checks the browser title is correct", () => {
+                commonPage
+                    .checkThatBrowserTitleForTrustPageMatches('Internal scrutiny reports - Financial documents - {trustName} - Find information about academies and trusts');
+            });
+
+            it("Checks the Internal scrutiny reports page subpage header is present", () => {
+                financialDocumentsPage
+                    .checkInternalScrutinyReportsPageHeaderPresent();
+            });
+
+            it("Checks the breadcrumb shows the correct page name", () => {
+                navigation
+                    .checkPageNameBreadcrumbPresent("Financial documents");
+            });
+
+            it("Checks the Internal scrutiny reports page financial status and year", () => {
+                financialDocumentsPage
+                    .checkInternalScrutinyReportCorrectStatusTypePresent()
+                    .checkFinancialDocumentsCorrectYearRangePresent();
+            });
+
+            it("Checks that the correct internal use only message is showing", () => {
+                financialDocumentsPage
+                    .checkForCorrectInternalUseMessage('Internal scrutiny reports are for internal use only.');
+            });
+
+        });
+
+        describe(`On the Self-assessment checklist page for a trust`, () => {
+
+            beforeEach(() => {
+                cy.login();
+                cy.visit(`/trusts/financial-documents/self-assessment-checklists?uid=${uid}`);
+            });
+
+            it("Checks the browser title is correct", () => {
+                commonPage
+                    .checkThatBrowserTitleForTrustPageMatches('Self-assessment checklists - Financial documents - {trustName} - Find information about academies and trusts');
+            });
+
+            it("Checks the Self assessment checklists page subpage header is present", () => {
+                financialDocumentsPage
+                    .checkSelfAssessmentChecklistsPageHeaderPresent();
+            });
+
+            it("Checks the breadcrumb shows the correct page name", () => {
+                navigation
+                    .checkPageNameBreadcrumbPresent("Financial documents");
+            });
+
+            it("Checks the Self assessment checklists page financial status and year", () => {
+                financialDocumentsPage
+                    .checkSelfAssessmentCorrectStatusTypePresent()
+                    .checkFinancialDocumentsCorrectYearRangePresent();
+            });
+
+            it("Checks that the correct internal use only message is showing", () => {
+                financialDocumentsPage
+                    .checkForCorrectInternalUseMessage('Self-assessment checklists are for internal use only.');
+            });
+
+        });
+
+        describe(`On the financial documents pages`, () => {
```

Comment:
This is nested under "Testing the Financial documents pages" so we should name it something different
```suggestion
        describe(`On every financial documents page`, () => {
```

Reply:
Now updated to the above